deploy/handler/operator.yaml

```diff
@@ -216,6 +216,13 @@ spec:
               mountPath: /var/k8s_nmstate
           securityContext:
             privileged: true
+          readinessProbe:
```

Comment:
IIUC what the [docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) say, we should specify `timeoutSeconds` explicitly, because in pre 1.20 version, the default 1 second timeout doesn't work and the probe processes run indefinitely until a command returns.

Maybe not a big risk in case of nmstatectl, but better be safe. wdyt @qinqon ?

Reply:
Good catch!!